src/components/Map/Map.module.css

```diff
@@ -0,0 +1,35 @@
+.container {
+  border-radius: var(--border-radius-m);
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.map {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: -30px;
+  top: 0;
+  outline: none;
+  box-shadow: var(--shadow);
+}
+
+.map * {
+  outline: none !important;
```

Comment:
Do you really need !important here? It's bad practice

Reply:
Done -> https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/0ccce385ccab27790268afdcb74876f0a3f7e79e